### ISSUE TITLE
Cannot add marks

### ISSUE THREAD

**Issue by paradox460** [brookhong/Surfingkeys#485]
Attempting to add any marks to any website fails.

Console:

```
common_content.min.js:4 Uncaught TypeError: Cannot read property 'scrollLeft' of null
    at Object.a.addVIMark (common_content.min.js:4)
    at common_content.min.js:4
a.addVIMark @ common_content.min.js:4
(anonymous) @ common_content.min.js:4
setTimeout (async)
a._handleMapKey @ common_content.min.js:4
(anonymous) @ common_content.min.js:4
eventListeners.(anonymous function) @ common_content.min.js:4
```

Seems to happen in the `addVIMark` function

**Comment by brookhong** [comment]
What are versions of your browser and Surfingkeys?

**Comment by durka** [comment]
I'm seeing this:

OSX 10.9.5
Chrome Version 60.0.3112.113 (Official Build) (64-bit)
Surfingkeys 0.9.8

**Comment by paradox460** [comment]
OS X 10.12.5  
Chrome 60.0.3112.113 (Official Build) (64-bit)  
Surfingkeys 0.9.8

**Comment by brookhong** [comment]
Thanks, fixed in 0.9.9.
